src/index.js

```diff
@@ -167,26 +168,32 @@ export default class TimeField extends React.Component {
   }
 
   render() {
-    const {value} = this.state;
-    const {onChange, style, showSeconds, input, colon, ...props} = this.props; //eslint-disable-line no-unused-vars
-    const onChangeHandler = (event) => this.onInputChange(event, (v) => onChange(v));
+    const { value } = this.state;
+    const { onChange, style, showSeconds, input, inputRef, colon, defaultValue, ...props } = this.props; //eslint-disable-line no-unused-vars
+    const onChangeHandler = (event) => this.onInputChange(event, (v) => onChange(event, v));
 
     if (input) {
       return React.cloneElement(input, {
         ...props,
+        colon,
         value,
         style,
-        onChange: onChangeHandler
+        defaultValue,
+        ref: inputRef,
+        onChange: onChangeHandler,
       });
     }
 
     return (
       <input
         type="text"
         {...props}
+        colon={colon}
+        defaultValue={defaultValue}
```

Comment:
The same there, as I see, there is no need to pass `defaultValue` separately, because all `...props` are already passed to the HTML input above (`{...props}` line 190)